config/crd/patches/webhook_in_initialbuildpipelineselectors.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
could be included directly in the crd file?

Reply:
Auto-generated. We don't need conversion webhook. Will delete it.